kolibri/auth/test/dummy_permissions_classes.py

```diff
@@ -0,0 +1,55 @@
+from ..permissions import BasePermissions
+
+
+class DenyAll(BasePermissions):
```

Comment:
I'm not quite sure I get the purpose of creating dummy classes here and in `dummy_test_models.py` --  in this case, what actually is being tested? If it's not ultimately testing application code then it is not useful. If you're trying to test some other application functionality that depends on these two somehow, it would be clearer (defined where it's used) and likely less verbose to use `mock`/`unittest.mock`.